docs/docs/03-tools/05-context.md

```diff
@@ -0,0 +1,64 @@
+# Context
+
+GPTScript supports context provider tools which can be used to provide additional context for tool execution. Output from the context tool gets prepended to the instruction of the current tool that uses the context tool.
```

Comment:
Can we document that exported tools from a context tools are added to the current tool's tools. (wow that was a bad way to say it).  Also can we document that you can just do `context: foo.txt` if you just want to add a string. This is a simple way to share prompts across tools.

Reply:
Following documentation is now added:
```
GPTScript provides a mechanism to share prompt information across many tools using the tool parameter `context`. It is used to provide additional information to the calling tool on when to use a specific tool by prepending the `context` to the instruction of the calling tool.

- Context can point to a static text or a GPTScript.
- Context tools are just regular GPTScript tools, and any valid gptscript field can be used.
- Exported tools from a context tools are available as tools to the calling tool.
- When context points to a GPTScript tool, output from the context tool gets prepended to the instruction of the calling tool.
```